README.md

```diff
@@ -672,6 +672,8 @@ Example configuration:
     //
     // The field is only effective if feature "trust-dns" is enabled.
     "dns": "google",
+    // Configure `cache_size` for "trust-dns" ResolverOpts. Set to "0" to disable DNS cache.
+    "dns_cache_size": "0",
```

Comment:
It should be a number. `"dns_cache_size": 0`

Reply:
Thanks! Sorry for my mistake ..

```suggestion
    "dns_cache_size": 0,
```